### DISCUSSION TITLE
proxy_protocol compatibility?

### DISCUSSION THREAD

**Discussion by joaogbcravo** [l7mp/stunner#100] [Q&A] [answered]
Hey!

Is stunner compatible with [proxy_protocol](https://www.haproxy.org/download/1.8/doc/proxy-protocol.txt)?

**Comment by levaitamas** [comment, marked as answer]
hi @joaogbcravo,
Currently not, but PRs are welcome. :smile:  STUNner implements the [TURN protocol](https://datatracker.ietf.org/doc/html/rfc8656).  For PROXY `nginx-ingress` might work: https://docs.nginx.com/nginx/admin-guide/load-balancer/using-proxy-protocol/
